mix.exs

```diff
@@ -8,11 +8,16 @@ defmodule Explorer.MixProject do
     [
       app: :explorer,
       name: "Explorer",
+      description: "Dataframes for Elixir.",
```

Comment:
```suggestion
      description: "Series (one-dimensional) and dataframes (two-dimensional) fast data exploration for Elixir",
```

Reply:
What  about "Series (one-dimensional) and dataframes (two-dimensional) __for__ fast data exploration for Elixir"?

Reply:
+1